src/applications/personalization/profile-2/sass/profile-2.scss

```diff
@@ -82,3 +82,37 @@ nav.profile-side-nav {
     margin: unset;
   }
 }
+
+
+.headerNameWrapper {
+  width: 60%;
+  h2 {
+    text-transform: capitalize;
+  }
+  @media (max-width: $medium-screen) {
+    width: 100%;
+    text-align: center;
+
+    h3 {
+      margin-top: 0;
+    }
+  }
+}
+
+.profileServiceBadge {
+  max-height: 108px;
+}
+
+.headerContentWrapper {
+  width: 100%;
+  @media (max-width: $medium-screen) {
```

Comment:
I think we can add the class `medium-screen:vads-u-flex-direction--column` instead 🙂

Reply:
Yeah, what Kelson said. Something along those lines, but doing `medium-screen:vads-u-flex-direction--column` _might_  set that for medium width or _larger_ screens, which is the opposite of what this rule does